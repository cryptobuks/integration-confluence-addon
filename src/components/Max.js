import React from 'react'
import Thumb from './Thumb'
import './max.css'

const Max = ({previewImgUrl, projectName, shareUrl}) => (
  <div className="wrapper">
    <Thumb
      previewImgUrl={previewImgUrl}
      projectName={projectName}
      shareUrl={shareUrl}
      maximized />
  </div>
)

export default Max
