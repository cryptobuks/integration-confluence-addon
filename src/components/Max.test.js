import React from 'react'
import renderer from 'react-test-renderer'
import Max from './Max'

it('renders correctly', () => {
  const previewImgUrl = 'https://s3.invisionapp-cdn.com/storage.invisionapp.com/screens/thumbnails/235857145.png?x-amz-meta-iv=2&response-cache-control=max-age%3D2419200&x-amz-meta-mdi=0&x-amz-meta-ck=7b6ac614880f2a6dcb831703094006aa&AWSAccessKeyId=AKIAJFUMDU3L6GTLUDYA&Expires=1506816000&Signature=xSZfe1UhbXL%2FbsWq%2F5jn%2FuhT0Fc%3D'
  const projectName = 'Paper embed prototypes'
  const shareUrl = 'https://in.invisionapp.com/share/ATBWCTHR5'
  const tree = renderer.create(
    <Max previewImgUrl={previewImgUrl} projectName={projectName} shareUrl={shareUrl} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
