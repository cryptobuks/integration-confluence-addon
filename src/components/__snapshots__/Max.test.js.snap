// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="wrapper"
>
  <a
    href="https://in.invisionapp.com/share/ATBWCTHR5"
    onClick={[Function]}
    rel="noopener noreferrer"
    target="_blank"
  >
    <img
      alt="Paper embed prototypes"
      className="thumb"
      onLoad={[Function]}
      src="https://s3.invisionapp-cdn.com/storage.invisionapp.com/screens/thumbnails/235857145.png?x-amz-meta-iv=2&response-cache-control=max-age%3D2419200&x-amz-meta-mdi=0&x-amz-meta-ck=7b6ac614880f2a6dcb831703094006aa&AWSAccessKeyId=AKIAJFUMDU3L6GTLUDYA&Expires=1506816000&Signature=xSZfe1UhbXL%2FbsWq%2F5jn%2FuhT0Fc%3D"
    />
  </a>
</div>
`;
